Python/Product/PythonTools/PythonTools/InterpreterList/InterpreterListToolWindow.cs

```diff
@@ -323,7 +323,6 @@ sealed class InterpreterListToolWindow : ToolWindowPane {
         }
 
         private void PipExtensionProvider_OperationStarted(object sender, OutputEventArgs e) {
-            _outputWindow.WriteLine(e.Data);
```

Comment:
Yes. We don't want to display the operation name. There'll be a specific message sent to display at the same time.

Sent from my Windows phone from an airport
________________________________
From: Hugues Valois<mailto:notifications@github.com>
Sent: ‎9/‎2/‎2016 10:47
To: Microsoft/PTVS<mailto:PTVS@noreply.github.com>
Cc: Steve Dower<mailto:Steve.Dower@microsoft.com>; Author<mailto:author@noreply.github.com>
Subject: Re: [Microsoft/PTVS] Fixes #1572 pip install shows double spacing within output (#1590)


In Python/Product/PythonTools/PythonTools/InterpreterList/InterpreterListToolWindow.cs<https://github.com/Microsoft/PTVS/pull/1590#discussion_r77385747>:

> @@ -323,7 +323,6 @@ sealed class InterpreterListToolWindow : ToolWindowPane {
>          }
>
>          private void PipExtensionProvider_OperationStarted(object sender, OutputEventArgs e) {
> -            _outputWindow.WriteLine(e.Data);


Is this WriteLine intentionally removed?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/Microsoft/PTVS/pull/1590/files/f822bb8232daa35a2ecad7da494f9ab8b446ca45#r77385747>, or mute the thread<https://github.com/notifications/unsubscribe-auth/ABnX-BUmNIAAYYNOdz1VXIeolM0lwE-Oks5qmGFBgaJpZM4Jz52u>.